docs/code-samples/getting-started/daml.yaml

```diff
@@ -40,6 +40,7 @@ data-dependencies:
   # - .lib/daml-finance-interface-instrument-generic.dar
   # - .lib/daml-finance-interface-instrument-option.dar
   # - .lib/daml-finance-interface-instrument-swap.dar
+  # - .lib/daml-finance-interface-types-date.dar
```

Comment:
need to reflect this in the `.template` file

Reply:
Seems to me it is already reflected?

Reply:
you are right. Sorry, my bad